src/app/redux/redux.module.ts

```diff
@@ -0,0 +1,42 @@
+import { NgModule } from '@angular/core';
+import { Store, createStore, compose, applyMiddleware } from 'redux';
+import { NgReduxModule, NgRedux, DevToolsExtension } from '@angular-redux/store';
+import { NgReduxFormModule, NgReduxFormConnectModule } from '@angular-redux/form';
+import thunk from 'redux-thunk';
+import { RootState, rootReducer, initialRootState } from '../store';
+
+import { PersistorService } from './persistor.service';
+
+@NgModule({
+  imports: [
+    NgReduxModule,
+  ],
+  exports: [
+    NgReduxModule,
+    NgReduxFormModule,
+    NgReduxFormConnectModule,
+  ],
+  providers: [ PersistorService ],
+})
+
+export class ReduxModule {
+  constructor(ngRedux: NgRedux<RootState>, persistor: PersistorService, devtools: DevToolsExtension) {
+
+    let store: Store<any>;
```

Comment:
I'm not familiar with setting up angular-redux, but I guess this makes sense. You might want to touch base with someone more familiar with ngRedux to make sure.